### ISSUE TITLE
Top-level readme missing 4th edition amd 2 2023 link

### ISSUE THREAD

**Issue by HinTak** [MPEGGroup/OpenFontFormat#58]
The top level README is missing links/references to 4th edition Amendment 2, 2023...

**Comment by vlevantovsky** [comment]
>The top level README is missing links/references to 4th edition Amendment 2, 2023...

Right, and the reason is the 4th edition Amendment 2 has not been made publicly available yet. I can provide the link to the [ISO main page for this document](https://www.iso.org/standard/82487.html) but that won't help anyone who wants to download it.
I've been trying to figure out the reasons for publication delays for quite some time now, so far have not heard anything back. Will keep trying and I am sure it will eventually be resolved.

**Comment by HinTak** [comment]
Is it normal for a delay in free download (vs a paid one) like, by a year? I don't remember having delays in the previous? But then I haven't followed anything before the 4th.

I got hold of some previews - apparently it is about 90 pages, a lot of it about COLRv1.
I guess I am not missing anything substantial/factual, as the WDs are online, and the MS OT spec is.

**Comment by vlevantovsky** [comment]
When it comes to dealing with ISO, the concept of "normal" does not apply. I have no idea why the amendment isn't publicly available yet, and the people I emailed about it did not respond.
You can use [colr spec source documents](https://github.com/googlefonts/colr-gradients-spec) as a reference, the differences between what was submitted vs. what's in the amendment are minor.
